src/ui/simulator/windows/inspector/frame.cpp

```diff
@@ -216,7 +216,7 @@ void Frame::onSelectAllPlantsFromArea(wxCommandEvent& evt)
         data->ThClusters.clear();
```

Comment:
**onSelectAllPlantsFromArea** : apparently not used (see comment before function), let's remove it